src/app/client/src/app/modules/core/services/user/user.service.ts

```diff
@@ -485,6 +487,14 @@ export class UserService {
       return response;
     }));
   }
+  
```

Comment:
@mithun30052001  @description - add in common place before the method  and fix the indentation

Reply:
Added in one common place @princegupta1131